ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-igmp-snooping.j2

```diff
@@ -1,23 +1,75 @@
-{% if ip_igmp_snooping is defined and ip_igmp_snooping is not none %}
+{% if ip_igmp_snooping is arista.avd.defined %}
 
 ## IP IGMP Snooping
 
 ### IP IGMP Snooping Summary
 
-{%     if ip_igmp_snooping.globally_enabled is defined and ip_igmp_snooping.globally_enabled == false %}
-IGMP snooping is globally disabled
-{%     else %}
-IGMP snooping is globally enabled.
+| Setting | Value |
```

Comment:
The documentation whould be much more compact and readable if we could make this into a regular table (only 1 row for global).
Same for querier (fine to keep querier in it's own table).

This is more important for vlans, where it will quickly grow to hundreds of lines just for igmp snooping if you have settings for many vlans. I think we should try to move vlans into a single table where each vlan is a row.